src/Test.FunctionalTests.BinSkim.Driver/CommandLineTests.cs

```diff
@@ -70,5 +70,40 @@ public void MostlyFunctionlessCommandlineTest()
             builder.Length.Should().Be(0,
                 $"all test cases should pass, but the following test cases failed:\n{builder}");
         }
+
+        [Fact]
+        public void DisableTelemetryCommandlineTest()
```

Comment:
>DisableTelemetryCommandlineTest [](http://example.com/codeflow?start=20&length=31)

i don't see any tests that prove that telemetry isn't collected in a tool run where this setting is specified. how do we know it works?